src/Pelagos/Bundle/AppBundle/Resources/views/Dataland/index.html.twig

```diff
@@ -42,6 +39,13 @@
 
     {{ add_js(path('fos_js_routing_js', { callback: 'fos.Router.setData' })) }}
 
+    {% if dataset.availabilityStatus == 10 %}
```

Comment:
Maybe put this in the place where {{parent()}} is, and put parent right under it, that way the META data is the first thing in the head. Technically doesn't matter. What do you think?

Reply:
I checked how it works now, it looks like its placed above, so i moved it 👍